### DISCUSSION TITLE
Microbit: Failed to load the MicroBit firmware to Micro.Bit V1.x board.

### DISCUSSION THREAD

**Discussion by robert-hh** [micropython/micropython#13746] [nRF]
A MicroBit v1.5 board refuses to accept the firmware provided on the MicroPython download site, and does as well not accept own built firmware. Firmware from the microbit.org site is accepted. The file sizes differ substantially. While the size of the microbit.org firmware is 273k, the one from micropyhon.org is 499k.
Did anyone make  a similar experience?

**Comment by rkompass** [comment]
I tried and was successfull in putting the latest firmware (1.22) from the MP downloads onto my Microbit 1.3 board.

The instructions at [the official Microbit site](https://microbit.org/get-started/user-guide/firmware/) may be wrong:
I read [here ??](https://tech.microbit.org/software/daplink-interface/#daplink-software) that in order to update the firmware you use the MICROBIT drive, in order to update the DAPLink you use the MAINTENANCE drive which appears after plugging the USB in with the button besides it pressed.
From this latter site I got the latest version of DAPLink which is 0253 now and installed it before trying the ordinary MP download.

Perhaps https://tech.microbit.org/software/hex-format/ is also be relevant, but I did not need it.
Hope this helps.

**Reply by robert-hh** [reply]
Thanks, it worked. You are right. The firmware has to be copied to the MICROBIT driver. I took the instructions from the Microbit page, which works with their version of the firmware. So maybe instructions have to be added to the MicroPython site.
